### PR TITLE
feat(components): add weather forecasts components

### PR DESCRIPTION
**Pull request by kyrylotaran**

### CODE REVIEW

File: src/assets/styles/variables/colors.scss [hunk 1]
```diff
@@ -1,2 +1,3 @@
 $white: #fff;
+$afro: #000;
```
Comment (vladsosnov) [inline src/assets/styles/variables/colors.scss]: Why afro? Maybe $black will be better?

File: src/components/WeatherForecasts/WeatherForecasts.module.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  width: 50px;
+  height: 50px;
+  background-color: rgba($afro, 0.3);
+  border: none;
+  border-radius: 50%;
+  cursor: pointer;
+
+  &::after {
+    display: inline-block;
+    padding: 4px;
```
Comment (vladsosnov) [inline src/components/WeatherForecasts/WeatherForecasts.module.scss:42]: $space-xs

Comment (kyrylotaran) [reply src/components/WeatherForecasts/WeatherForecasts.module.scss:42]: we don't use spaces cariables for paddings cause we would have too many variables for different padding cases

File: src/assets/styles/variables/colors.scss (hunk 1 above)
Comment (kyrylotaran) [reply src/assets/styles/variables/colors.scss]: yeap

**Review by vladsosnov — DISMISSED** [review]

**Review by vladsosnov — APPROVED** [review]

**Pull request merged**